03_letters/madhavansu/main_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var tests = []struct {
+	s    string
```

Comment:
Try to use more descriptive variables. 
`s` -> `input`, `in`,
`lmap` -> `frequencyMap`, `letterMap`
`ss` -> `output`, `sorted`

Reply:
Updated.